wandb/sdk/artifacts/artifact.py

```diff
@@ -2422,19 +2422,14 @@ def _delete(self, delete_aliases: bool = False) -> None:
 
     @normalize_exceptions
     def link(self, target_path: str, aliases: Iterable[str] | None = None) -> Artifact:
-        """Link this artifact to a portfolio (a promoted collection of artifacts).
+        """Link this artifact to a collection.
 
         Args:
-            target_path: The path to the portfolio inside a project.
-                The target path must adhere to one of the following
-                schemas `{portfolio}`, `{project}/{portfolio}` or
-                `{entity}/{project}/{portfolio}`.
-                To link the artifact to the Model Registry, rather than to a generic
-                portfolio inside a project, set `target_path` to the following
-                schema `{"model-registry"}/{Registered Model Name}` or
-                `{entity}/{"model-registry"}/{Registered Model Name}`.
-            aliases: Optional string aliases that will uniquely identify the artifact
-                inside the specified portfolio.
+            target_path: The path of the collection. Path consists of the prefix
+                "wandb-registry-" along with the registry name and the
+                collection name `wandb-registry-{REGISTRY_NAME}/{COLLECTION_NAME}`.
+            aliases: Add one or more aliases to the linked artifact. Apply
+                the alias "latest" to the latest artifact version that is linked.
```

Comment:
I dont think we should mention applying the `latest` alias since that automatically gets applied